### PR TITLE
Add list command

### LINKED ISSUE #14: Improve CLI DX with list in shell, for link app, version

### PR DESCRIPTION
**Pull request by arendjantetteroo**
Fixes #14

### CODE REVIEW

**Comment by arendjantetteroo** [comment]
@riderx A first attempt at a list command. Let me know if this is acceptable or needs more changes?

File: src/bin/delete.ts [hunk 1]
```diff
@@ -20,20 +20,11 @@ export const deleteApp = async (appid: string, options: Options) => {
   if (!appid) {
     program.error('Missing argument, you need to provide a appid, or be in a capacitor project');
   }
-  console.log(`Delete ${appid} to Capgo`);
+  console.log(`Delete ${appid} - ${version} from Capgo`);
```
Comment (riderx) [inline src/bin/delete.ts:23]: version here will not be always present since it's optional

**Comment by riderx** [comment]
Hey @arendjantetteroo thank you so much for the work! it's excellent!
if you can just fix the console.log issue i commented, then i merge :)

**Comment by riderx** [comment]
Ok no worry, i merged and improved the logs :)

**Pull request merged**